### PR TITLE
Tarea 2: Semantic HTML

### PR DESCRIPTION
**Pull request by s-rendong**

### CODE REVIEW

File: sebastian-rendon/tarea-2/index.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+      >
+        Sept. 4, 2022 - Ed. 42
+      </span>
+      <header class="header">
+        <h1 title="Checkered flag" class="header__magazine-title">
+          <span>checkered</span>
+          <span>flag</span>
+        </h1>
+      </header>
+      <img
```
Comment (emolinaab) [inline sebastian-rendon/tarea-2/index.html:28]: @s-rendong you should figure tag for all images

File: sebastian-rendon/tarea-2/index.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            <h1>Oscar Piastri</h1>
+            <h2>Who is the new McLaren driver?</h2>
+          </header>
+          <p>
+            The Australian promise talks about his expectations with the
+            Woking-based team.
+          </p>
+        </article>
+        <aside class="aside__supporting-articles">
+          <h1 class="title__supporting-article--first">
```
Comment (emolinaab) [inline sebastian-rendon/tarea-2/index.html:47]: @s-rendong , there should be only one h1 tag in a document

File: sebastian-rendon/tarea-2/index.html [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    <meta
+      name="description"
+      content="Mock Magazine Cover about Oscar Piastri"
+    />
+    <link rel="stylesheet" href="./styles.css" />
+    <title>Magazine</title>
+  </head>
+  <body>
+    <main class="main__magazine">
+      <span
```
Comment (emolinaab) [inline sebastian-rendon/tarea-2/index.html:16]: @s-rendong, research about the use of date tag

**Review by emolinaab — CHANGES_REQUESTED** [review]

File: sebastian-rendon/tarea-2/index.html (hunk 2 above)
Comment (s-rendong) [reply sebastian-rendon/tarea-2/index.html]: fixed with [[8393c42](https://github.com/emolinaab/bootcamp-001/pull/88/commits/8393c4295e96ea5b6bb1fd2804e2903da42f3180)](https://github.com/emolinaab/bootcamp-001/pull/88/commits/8393c4295e96ea5b6bb1fd2804e2903da42f3180)

File: sebastian-rendon/tarea-2/index.html (hunk 3 above)
Comment (s-rendong) [reply sebastian-rendon/tarea-2/index.html]: fixed with https://github.com/emolinaab/bootcamp-001/pull/88/commits/8393c4295e96ea5b6bb1fd2804e2903da42f3180

File: sebastian-rendon/tarea-2/index.html (hunk 1 above)
Comment (s-rendong) [reply sebastian-rendon/tarea-2/index.html]: fixed with https://github.com/emolinaab/bootcamp-001/pull/88/commits/8393c4295e96ea5b6bb1fd2804e2903da42f3180

**Review by emolinaab — APPROVED** [review]

**Pull request merged**